dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -1177,3 +1172,20 @@ CWWKD1116.resource.unavailable.explanation=A resource that the repositories \
  require did not become available.
 CWWKD1116.resource.unavailable.useraction=Verify the configuration of resources \
  that the repositories use.
+
+CWWKD1117.anno.constraint.conflict=CWWKD1117E: Parameter {0} of the {1} method \
+ of the {2} repository interface cannot be annotated with the annotations: {3}. \
+ One or more of the annotations is incompatible with the {4} type of the \
+ repository method parameter.
+CWWKD1117.anno.constraint.conflict.explanation=Constraint and its subtypes \
+ cannot be combined with parameter annotations other than the By annotation.
+CWWKD1117.anno.constraint.conflict.useraction=Remove the conflicting annotation \
+ or specify the entity attribute type instead of a Constraint type.
+
+CWWKD1118.param.anno.conflict=CWWKD1118E: Parameter {0} of the {1} method \
+ of the {2} repository interface cannot be annotated with the annotations: {3}. \
```

Comment:
Instead of `Parameter {0}`, use `the {0} parameter` unless the parameter name is hard to read.

Alternatively, to simplify "cannot be annotated," change the entire first sentence to: "Do not annotate the {0} parameter of the {1} method of the {2} repository interface with the annotations: {3}."

Reply:
Same reason here for parameter number vs name. I'll look into making the other change.

Reply:
After looking into the second change, I believe it will result in ambiguity.  One could misread "with the annotations" as referring to the repository interface. (These can also have annotations).  The intent is for "with the annotations" to refer to the parameter.  That was clear in the original text, but not after the update, so I think this will need to remain as is as well.